docs/deployment-key-rotation.md

```diff
@@ -3,25 +3,26 @@ Martha Deployment Key Rotation
 
 The service account key used to deploy Martha needs to be rotated every 90 days.
 
-1. Find the current BT ticket for rotating the deployment SA key. If these instructions were followed, it will be in the Slack notification that led you here. Add it to the current Jira board if it's not already there.
-2. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng). Not strictly necessary, but this is expected to pass and sets a current baseline to avoid surprises if the post-rotation test deploy fails.
-3. Using your `@firecloud.org` account, [create a new key for the `cloud-functions-account@broad-dsde-prod.iam.gserviceaccount.com` service account](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod).
+1. Find the current WX ticket for rotating the deployment SA key. If these instructions were followed, it will be in the Slack notification that led you here. Add it to the current Jira board if it's not already there.
+2. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng). This is expected to pass and sets a current baseline to avoid surprises if the post-rotation test deploy fails.
+3. Disable all existing key(s) on [this list](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod). Using your @firecloud.org account:
```

Comment:
According to the docs it is straightforward to re-enable one of these keys if necessary. That makes me feel even better about this change.

Reply:
Yeah my thinking was that it's important to disable the old keys to ensure the test isn't accidentally using one of them. And then once the test passes, they can be safely deleted.